spec/examples.txt

```diff
@@ -0,0 +1,8 @@
+example_id                                      | status | run_time        |
```

Comment:
Not sure where this file is coming from, but we don't need to track in in Version Control. I suggest we delete it and add to `.gitignore`

Reply:
WIP! Correction coming soon.

For the same reason, I didn't change this either.

There is another important correction.
The name of the application is `DgmsNewsAdmin` - DgmsNews**Admin**, and it should be `DgmsNewsApi` - DgmsNews**Api**.
Should I change that?